### DISCUSSION TITLE
Game crashes when going to a custom week

### DISCUSSION THREAD

**Discussion by TheCapitalist** [KadeArchive/Kade-Engine#584] [Q&A] [answered]
Im putting this here because I dont know if it is a bug or if im stupid.
While making a mod I tried to play my week to check dialogue, and it crashed even before selecting the week, I tried to disable the dialogue, still didnt work, yet freeplay works fine, I did everything correctly and it still crashes, Im using Kade Engine 1.4.2

**Comment by puyoxyz** [comment]
did you get an error in the commandline output

**Reply by TheCapitalist** [reply]
No, it traced 8 lines in debug then a window shows up saying "Null object reference" like this 
![image](https://user-images.githubusercontent.com/83668596/119660998-2138a480-be30-11eb-9e46-5f5708c2390c.png)

**Comment by isakube** [comment]
Mandatory question: Were you following the Week Guide on the KE website?

Also, you might want to add in the storymenu image if you didn't already, if you did already then huh

**Reply by TheCapitalist** [reply]
When I added the image, still didnt work and yes, I did follow the week guide

**Comment by TheCapitalist** [comment, marked as answer]
after some digging I noticed that I forgot to make the character sprites and the game couldn't find it so it was crashing
